website/docs/r/rds_instance_v3.html.markdown

```diff
@@ -0,0 +1,259 @@
+---
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_rds_instance_v3"
+sidebar_current: "docs-opentelekomcloud-resource-rds-instance-v3"
+description: |-
+  instance management
+---
+
+# opentelekomcloud\_rds\_instance\_v3
+
+instance management
+
+## Example Usage
+
+### create a single db instance
+
+```hcl
+resource "opentelekomcloud_networking_secgroup_v2" "secgroup" {
+  name = "terraform_test_security_group"
+  description = "terraform security group acceptance test"
+}
+
+resource "opentelekomcloud_rds_instance_v3" "instance" {
+  availability_zone = "{{ availability_zone }}"
+  db = {
+    password = "Huangwei!120521"
+    type = "PostgreSQL"
+    version = "9.5"
+    port = "8635"
+  }
+  name = "terraform_test_rds_instance"
+  security_group_id = "${opentelekomcloud_networking_secgroup_v2.secgroup.id}"
+  network_id = "{{ network_id }}"
+  vpc_id = "{{ vpc_id }}"
+  volume {
+    type = "COMMON"
+    size = 100
+  }
+  flavor = "rds.pg.c2.medium"
+  backup_strategy = {
+    start_time = "08:00-09:00"
+    keep_days = 1
+  }
+}
+```
+
+### create a primary/standby db instance
+
+```hcl
+resource "opentelekomcloud_networking_secgroup_v2" "secgroup" {
+  name = "terraform_test_security_group"
+  description = "terraform security group acceptance test"
+}
+
+resource "opentelekomcloud_rds_instance_v3" "instance" {
+  availability_zone = "{{ availability_zone_1 }},{{ availability_zone_2 }}"
+  db = {
+    password = "Huangwei!120521"
+    type = "PostgreSQL"
+    version = "9.5"
+    port = "8635"
+  }
+  name = "terraform_test_rds_instance"
+  security_group_id = "${opentelekomcloud_networking_secgroup_v2.secgroup.id}"
+  network_id = "{{ network_id }}"
+  vpc_id = "{{ vpc_id }}" 
+  volume {
+    type = "COMMON"
+    size = 100
+  }
+  flavor = "rds.pg.s1.medium.ha"
+  ha_replication_mode = "async"
+  backup_strategy = {
+    start_time = "08:00-09:00"
+    keep_days = 1
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `availability_zone` -
+  (Required)
+  Specifies the AZ name. Changing this parameter will create a new resource.
+
+* `db` -
+  (Required)
+  Specifies the database information. Structure is documented below. Changing this parameter will create a new resource.
+
+* `flavor` -
```

Comment:
Is  the argument `flavor` replacing the `flavorref` argument from v1? 


Is the following example working?
```hcl
data "opentelekomcloud_rds_flavors_v3" "flavor" {
    db_type = "PostgreSQL"
    db_version = "9.5"
    instance_mode = "ha"
}

resource "opentelekomcloud_rds_instance_v3" "instance" {
  name = "terraform_test_rds_instance"
  ...
  flavor = "${data.opentelekomcloud_rds_flavors_v3.flavor.name}"
  ...
}
```

Reply:
for v1 api, the flavorref is the id of flavor. but for v3, flavorref  is the name of it.  you can not use the data source of opentelekomcloud_rds_flavors_v3 like that. because data.opentelekomcloud_rds_flavors_v3.flavor is an array, it just show all the flavors filtered by specified flavor information.

Reply:
So the recommended way is to specify the flavor with the spec code directly in the rds_v3 resource like in the example below?

```
resource "opentelekomcloud_rds_instance_v3" "instance" {
  name = "terraform_test_rds_instance"
  ...
  flavor = "rds.pg.c2.medium.ha"
  ...
}
```

Reply:
yes

Reply:
Perfect. This makes the configuration simpler. 👍